### PR TITLE
Microsoft.Management 2020-05-01 apiversion

### PR DESCRIPTION
**Pull request by ramoka178**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [x] Service team MUST add the "**WaitForARMFeedback**" label if the management plane API changes fall into one of the below categories. 
  - adding/removing APIs.
  - adding/removing properties.
  - adding/removing API-version. 
  - adding a new service in Azure.

<i>Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.</i>

- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by ramoka178** [comment]
@xccc-msft : Any guidance that you could provide on the 2 failing checks in SDK Java? I looked through the error, but its pointing to source code from another repo, probably something from SDK Java?!

File: specification/managementgroups/resource-manager/Microsoft.Management/stable/2020-05-01/examples/CheckManagementGroupNameAvailability.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "parameters": {
+    "api-version": "2020-05-01",
+    "checkNameAvailabilityRequest": {
+      "name": "nameTocheck",
+      "type": "Microsoft.Management/managementGroup"
```
Comment (stivanov-git) [inline specification/managementgroups/resource-manager/Microsoft.Management/stable/2020-05-01/examples/CheckManagementGroupNameAvailability.json:6]: It should be "Microsoft.Management/managementGroups" type

Comment (ramoka178) [reply specification/managementgroups/resource-manager/Microsoft.Management/stable/2020-05-01/examples/CheckManagementGroupNameAvailability.json:6]: "CheckNameAvailabilityRequest" definition was pointing to  "Microsoft.Management/managementGroup", and the validation was throwing an error.
Will make the change in the definition and revert this change in next iteration

**Review by stivanov-git — APPROVED** [review]
Just a small change in one of the files is still needed. Approved

File: specification/managementgroups/resource-manager/Microsoft.Management/stable/2020-05-01/examples/GetAllSubscriptionsFromManagementGroup.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    "api-version": "2020-05-01",
+    "groupId": "Group"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "name": "728bcbe4-8d56-4510-86c2-4921b8beefbc",
+            "id": " /providers/Microsoft.Management/managementGroups/Group/subscriptions/728bcbe4-8d56-4510-86c2-4921b8beefbc",
```
Comment (chiragg4u) [inline specification/managementgroups/resource-manager/Microsoft.Management/stable/2020-05-01/examples/GetAllSubscriptionsFromManagementGroup.json:12]: >  [](start = 19, length = 1)

Extra space here.

File: specification/managementgroups/resource-manager/Microsoft.Management/stable/2020-05-01/examples/GetAllSubscriptionsFromManagementGroup.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+            "type": "Microsoft.Management/managementGroups/subscriptions",
+            "properties": {
+              "displayName": "Group",
```
Comment (chiragg4u) [inline specification/managementgroups/resource-manager/Microsoft.Management/stable/2020-05-01/examples/GetAllSubscriptionsFromManagementGroup.json:15]: >displayName": "Group", [](start = 15, length = 22)

Is this subscription display name?

File: specification/managementgroups/resource-manager/Microsoft.Management/stable/2020-05-01/examples/GetAllSubscriptionsFromManagementGroup.json [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+              "parent": {
+                "id": "/providers/Microsoft.Management/managementGroups/Group"
+              },
+              "state": "Active",
```
Comment (chiragg4u) [inline specification/managementgroups/resource-manager/Microsoft.Management/stable/2020-05-01/examples/GetAllSubscriptionsFromManagementGroup.json:19]: >"state": "Active", [](start = 14, length = 18)

How do we keep these properties synced?

File: specification/managementgroups/resource-manager/Microsoft.Management/stable/2020-05-01/examples/GetAllSubscriptionsFromManagementGroup.json (hunk 2 above)
Comment (ramoka178) [reply specification/managementgroups/resource-manager/Microsoft.Management/stable/2020-05-01/examples/GetAllSubscriptionsFromManagementGroup.json]: Resolved

File: specification/managementgroups/resource-manager/Microsoft.Management/stable/2020-05-01/examples/GetAllSubscriptionsFromManagementGroup.json (hunk 3 above)
Comment (ramoka178) [reply specification/managementgroups/resource-manager/Microsoft.Management/stable/2020-05-01/examples/GetAllSubscriptionsFromManagementGroup.json]: Yes, it should be, changed now

File: specification/managementgroups/resource-manager/Microsoft.Management/stable/2020-05-01/examples/GetAllSubscriptionsFromManagementGroup.json (hunk 4 above)
Comment (ramoka178) [reply specification/managementgroups/resource-manager/Microsoft.Management/stable/2020-05-01/examples/GetAllSubscriptionsFromManagementGroup.json:19]: Management Groups receives subscription notifications from ARM and Provisioning team if there are any changes to Subscription state

**Review by xseeseesee — APPROVED** [review]

**Pull request merged**